index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <title>Editor Staff Hub</title>
+  <meta charset="utf-8">
+  <meta name="author" content="Editor Staff Hub j4502">
+  <meta name="description" content="Editor Staff Hub">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+ </head>
+
+<body>
+<img src="assets/images/logo.png" alt="Columbia Missourian Logo">
```

Comment:
Good job of thinking through how to grab the weather icon and the Missourian logo and linking them up.